packages/kbn-unified-data-table/src/components/data_table.test.tsx

```diff
@@ -846,4 +846,24 @@ describe('UnifiedDataTable', () => {
       expect(getFieldColumns()).toEqual(['@timestamp', '_index', 'bytes', 'extension', 'message']);
     });
   });
+
+  describe('getRowColorIndicator', () => {
+    it('should render the color indicator control', async () => {
+      const component = await getComponent({
+        ...getProps(),
+        getRowIndicatorColor: jest.fn(() => 'blue'),
+      });
+
+      expect(findTestSubject(component, 'dataGridHeaderCell-colorIndicator').exists()).toBeTruthy();
```

Comment:
Once we've settled on a message for screen readers, could we add a test asserting that message string appears in the DOM?

Reply:
Added via https://github.com/elastic/kibana/pull/186418/commits/b903ba1347dc4a81e28902a34bacbf7c24a9c997